CHANGELOG.md

```diff
@@ -10,114 +10,121 @@ The following classes, methods, properties and events have been deprecated for a
 
 | Class/Property/Method/Event | Alternate option | Version deprecated |
 |----------|-------------|--------------------|
+| `BasemapLayerList.statusIndicatorsVisible` | `BasemapLayerList.visibleElements.statusIndicators` | 4.15 |
 | `Bookmarks.bookmarkCreationOptions` | `Bookmarks.defaultCreateOptions` | 4.18 |
+| `esri/tasks/ImageServiceIdentifyTask` | `imageService.identify` | 4.18 |
+| `esri/tasks/support/ImageServiceIdentifyResult` | `ImageIdentifyResult` | 4.18 |
+| `esri/tasks/support/ImageServiceIdentifyParameters` | `ImageIdentifyParameters` | 4.18 |
+| `FeatureForm.description` and `FeatureFormViewModel.description`  | `FormTemplate.description` set either on the `FeatureForm` or `FeatureLayer.formTemplate` | 4.18 |
+| `FeatureForm.fieldConfig` and `FeatureFormViewModel.fieldConfig` | `FieldElement/GroupElement` set in the `FormTemplate.elements` | 4.16 |
+| `FeatureForm.title` and `FeatureFormViewModel.title` | `FormTemplate.title` set either on the `FeatureForm` or `FeatureLayer.formTemplate`  | 4.18 |
```

Comment:
I would remove these. they're breaking changes in 4.21 not 4.22

Reply:
SGTM.